mithril-common/src/crypto_helper/era.rs

```diff
@@ -30,57 +30,40 @@ pub struct EraMarkersSigner {
 }
 
 impl EraMarkersSigner {
-    /// EraMarkersSigner factory
+    /// [EraMarkersSigner] factory
     pub fn create_test_signer<R>(mut rng: R) -> Self
     where
         R: CryptoRng + RngCore,
     {
-        let secret_key = ed25519_dalek::SecretKey::generate(&mut rng);
+        let secret_key = ed25519_dalek::SigningKey::generate(&mut rng);
```

Comment:
nit: secret_key => signing_key

Reply:
We choose to keep the "secret key" name, since it's the same naming scheme that cardano use for those kind of keys.

Reply:
Sorry for chiming in but actually that's not true: If you search for `"secret key"` in https://docs.cardano.org/ you won't find anything, but if you search for `"signing key"` you'll find multiple occurrences. In the code, the different types of keys are indeed `SigningKey` and `VerificationKey`, eg. [here](https://github.com/input-output-hk/cardano-base/blob/master/cardano-crypto-class/src/Cardano/Crypto/DSIGN/Class.hs#L87) or [here](https://github.com/input-output-hk/cardano-api/blob/aaba33e50116957bbf7697c2cfe162d15d92a7b1/cardano-api/internal/Cardano/Api/Keys/Class.hs#L45). 
Your context might be different though.

Reply:
Oh shoot ! We were so convinced that it was the case that we did not "fact check" our assumption.
We will do the change in another PR since the "secret" word is used quite a lot in our code (10+ in our rust code alone, but not all have to be updated).